leaderelection/ordinal.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leaderelection
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const controllerOrdinalEnv = "CONTROLLER_ORDINAL"
+
+// ControllerOrdinal tries to get ordinal from the pod name of a StatefulSet,
```

Comment:
Is this going to be used outside of the package?

Reply:
I think a downstream pod, autoscaler for example, needs to know its own ordinal so that it can compare whether a bucket belongs to itself. The relationship introduced in https://gist.github.com/vagababov/47e5f62c5865dbcc9deb1cd48655d7d2 is `key -> bucket ordinal`. We need to compare bucket ordinal and pod ordinal.

Reply:
No that's fine, I just wanted to make sure _it is_ going to be used.